src/ontology/obi-edit.owl

```diff
@@ -21882,6 +21917,645 @@ b) that the amount of analyte/measurand detected is over some predetermined thre
+        <obo:IAO_0000117>https://orcid.org/0000-0002-7245-3450 &quot;Lauren M. Wishnie&quot;</obo:IAO_0000117>
+        <obo:IAO_0000117>https://orcid.org/0000-0002-9821-4132 &quot;Mackenzie T. Smith&quot;</obo:IAO_0000117>
+        <obo:IAO_0000118>scan</obo:IAO_0000118>
+        <rdfs:label xml:lang="en">reconstructed MR image data set</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/OBI_0003335 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/OBI_0003335">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/OBI_0003334"/>
+        <obo:IAO_0000115>A type of MR image data set collected using at least three diffusion gradients of some factor &apos;b,&apos; which is a function of the amplitude, duration, and interval of the gradiants.</obo:IAO_0000115>
+        <obo:IAO_0000117>https://orcid.org/0000-0001-9676-7377 &quot;Alexander D. Bartnik&quot;</obo:IAO_0000117>
+        <obo:IAO_0000117>https://orcid.org/0000-0001-9990-8331 &quot;Alexander D. Diehl&quot;</obo:IAO_0000117>
+        <rdfs:label xml:lang="en">diffusion weighted image data set</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/OBI_0003336 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/OBI_0003336">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/OBI_0003334"/>
+        <obo:IAO_0000115>An MR image data set collected as a 4D time series representing changes in brain metabolism (i.e. BOLD), typically used as a proxy for brain activity.</obo:IAO_0000115>
+        <obo:IAO_0000117>https://orcid.org/0000-0001-9625-1899 &quot;William D. Duncan&quot;</obo:IAO_0000117>
+        <obo:IAO_0000117>https://orcid.org/0000-0001-9676-7377 &quot;Alexander D. Bartnik&quot;</obo:IAO_0000117>
+        <obo:IAO_0000117>https://orcid.org/0000-0001-9990-8331 &quot;Alexander D. Diehl&quot;</obo:IAO_0000117>
+        <obo:IAO_0000117>https://orcid.org/0000-0002-2104-0568 &quot;Lucas M. Serra&quot;</obo:IAO_0000117>
+        <rdfs:label xml:lang="en">functional MR image data set</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/OBI_0003337 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/OBI_0003337">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/OBI_0003336"/>
+        <obo:IAO_0000115>A type of functional MRI in which the participant is not given a task, but just rests during acquisision</obo:IAO_0000115>
+        <obo:IAO_0000117>https://orcid.org/0000-0001-9676-7377 &quot;Alexander D. Bartnik&quot;</obo:IAO_0000117>
+        <obo:IAO_0000117>https://orcid.org/0000-0001-9990-8331 &quot;Alexander D. Diehl&quot;</obo:IAO_0000117>
+        <obo:IAO_0000117>https://orcid.org/0000-0002-2104-0568 &quot;Lucas M. Serra&quot;</obo:IAO_0000117>
+        <rdfs:label xml:lang="en">Resting state functional MR image data set</rdfs:label>
```

Comment:
OBI uses lower case labels, so "Resting" should be "resting". We also avoid acronyms in primary labels, so I think that "MR" should be expanded. Sometimes exceptions are made for clarity and length.